package.json

```diff
@@ -74,13 +69,13 @@
     "calculate-cache-key-for-tree": "^2.0.0",
     "cldr-core": "^43.1.0",
     "ember-auto-import": "^2.6.3",
-    "ember-cli-babel": "^7.26.11",
+    "ember-cli-babel": "^8.0.0",
     "ember-cli-typescript": "^5.2.1",
-    "eventemitter3": "^4.0.7",
+    "eventemitter3": "^5.0.1",
```

Comment:
There shouldn't be breaking changes that affect us.

https://github.com/primus/eventemitter3/releases/tag/5.0.0